renovate.json

```diff
@@ -48,6 +48,20 @@
       "groupName": "Vaul",
       "matchDepNames": ["vaul"],
       "prHeader": "Please review modals on mobile for visual regressions."
+    },
+    {
+      "groupName": "bundled element-call dependencies",
```

Comment:
Perhaps we could stick to the term "embedded" here?

Reply:
I dont like embedded. It is what we use for `embedded.js` and spa vs widget in general so I think it might lead to confusion to overload the word?

But maybe this is a discussion for a meeting.
wdyt?